### PR TITLE
Build Result page

### PR DESCRIPTION
**Pull request by ishitakeshawani**
Preview link : https://deploy-preview-4--memorynomads.netlify.app/resultpage/resultpage

### CODE REVIEW

**Review by ckahub — COMMENTED** [review]
lgtm

File: Resultpage/resultpage.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.result-page {
+  margin: 6rem 2rem 2rem 2rem;
```
Comment (hiteshverma27) [inline Resultpage/resultpage.css:2]: suggestion- you can make utility classes for this

**Review by hiteshverma27 — APPROVED** [review]
Looks good to me

File: Resultpage/resultpage.css (hunk 1 above)
Comment (ishitakeshawani) [reply Resultpage/resultpage.css:2]: Yes.

**Pull request merged**